### DISCUSSION TITLE
Hilfe gesucht

### DISCUSSION THREAD

**Discussion by blackpearlla** [jomjol/AI-on-the-edge-device#1697] [Q&A]
Hi leute, erstmal will ich mich beim Entwickler bedanken für diese Großartige Lösung. 

Dennoch habe ich ein kleinen Problem mit meinem Wasserzähler.. Bitte schaut euch das Bild an und helft mir auf die Sprünge.. die Digits und die erste "Uhr" sind natürlich kein Problem. Aber wie dann weiter? Gibt es die Möglichkeit an dritter stelle ein Digit zu verwenden?

![image](https://user-images.githubusercontent.com/11446500/209675181-f1344e6b-3da9-4689-b94c-3319de9883ed.png)

Thanks for help!

**Comment by Slider0007** [comment]
Hallo, ich habe beim letzten Wechsel den gleichen Zählertyp aufs Auge gedrückt bekommen.
Ich verwende nur ana1 und ana2 und habe  extendendResolution aktiviert. Somit ergeben sich 3 Nachkommastellen.

**Reply by blackpearlla** [reply]
Liest er den Ana2 auch richtig aus? Das kann ich mir fast nicht vorstellen.

**Reply by Slider0007** [reply]
Das funktioniert bei mir echt ohne jegliche Probleme. Ich war Anfangs auch skeptisch...

**Comment by Esp2use** [comment]
Zählt bei Euch der ANA1 richtig, bzw. wieviel mal absolviert der ANA1 die Runde, bis der rechte Digit komplett gewechselt hat? 1x oder dreht er bei Euch auch ANA1 2x die Runde? Sieht auf dem Foto zumindest mal so aus. 1 Runde ANA1 wären doch rein theoretisch 1m³. Wäre aber in diesem Fall wohl nicht so...

**Reply by Slider0007** [reply]
Jeder analoge Zeiger dreht jeweils einmal. Durch die Mechanik ergeben sich aber jeweils leichte Varianzen. Dieser Zählertyp zieht die Digits relativ spät nach.
Bei mir funktioniert die Erkennung fast zu 100%
